### PR TITLE
Feature controllers

### PR DESCRIPTION
**Pull request by Gabriel-B12**
Two controllers added and some minor changes

### CODE REVIEW

File: src/util/database.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 const username = "projectDB";
 const password = "projectDBpass";
 const cluster = "cluster0.4jlsc";
 const dbname = "projectDB";
 export const dbURL = `mongodb+srv://${username}:${password}@${cluster}.mongodb.net/?retryWrites=true&w=majority`;
-*/
+
     
-export const dbURL = `mongodb://${MONGO_HOSTNAME}:${MONGO_PORT}`;
+// export const dbURL = `mongodb://${MONGO_HOSTNAME}:${MONGO_PORT}`;
```
Comment (VladMRusu1) [inline src/util/database.js:12]: I think you don't need this comment anymore

File: src/controllers/events.js [hunk 2]
```diff
@@ -85,12 +85,58 @@ export const postEvent = async (req, res, next) => {
...
 };
 
-export const getEvent = async (_req, res, next) => {
+export const getEvent = async (req, res, next) => {
+  try {
+    const id = req.params.id;
+    if (!mongoose.Types.ObjectId.isValid(id))
+      throw new ProblemError(
+        MESSAGE_TYPES.ERROR,
+        400,
```
Comment (VladMRusu1) [inline src/controllers/events.js:94]: Maybe all error codes you can move in consts to be easier to change it if you want another error code for a specific error. Is just a suggestion is not wrong like this

**Review by VladMRusu1 — APPROVED** [review]
Looks good, just 2 suggestions

**Review by cristicirdei — COMMENTED** [review]
🗸

**Pull request merged**